cliRoutes/CustomSearch/index.js

```diff
@@ -36,68 +36,34 @@ module.exports = class SearchOnline {
                   switch (confirmation) {
                     case 'Yes':
                       prompt(YarnOrNpm).then(({ packageManager }) => {
-                        switch (packageManager) {
-                          case 'Yarn':
-                            prompt(GlobalOrNot).then(({ confirm }) => {
-                              switch (confirm) {
-                                case 'Yes':
-                                  customCMD.get(
-                                    `yarn add ${this.packagesSelected.join(' ')} -g`,
-                                    'install',
-                                    `Package(s): ${this.packagesSelected.join(
-                                      ' '
-                                    )} have been installed successfully!`
-                                  )
-                                  break
-
-                                case 'No':
-                                  customCMD.get(
-                                    `yarn add ${this.packagesSelected.join(' ')}`,
-                                    'install',
-                                    `Package(s): ${this.packagesSelected.join(
-                                      ' '
-                                    )} have been installed successfully!`
-                                  )
-                                  break
-                              }
-                            })
-                            break
-
-                          case 'NPM':
-                            prompt(GlobalOrNot).then(({ confirm }) => {
-                              switch (confirm) {
-                                case 'Yes':
-                                  customCMD.get(
-                                    `npm install ${this.packagesSelected.join(' ')} -g`,
-                                    'install',
-                                    `Package(s): ${this.packagesSelected.join(
-                                      ' '
-                                    )} have been installed successfully!`
-                                  )
-                                  break
-
-                                case 'No':
-                                  customCMD.get(
-                                    `npm install ${this.packagesSelected.join(' ')}`,
-                                    'install',
-                                    `Package(s): ${this.packagesSelected.join(
-                                      ' '
-                                    )} have been installed successfully!`
-                                  )
-                                  break
-                              }
-                            })
-                          break
-                        }
+                        prompt(GlobalOrNot).then(({ confirm }) => {
+                          customCMD.get(
+                            `${
```

Comment:
Some of this ternary logic is reused quite a lot, can turn it into a func

Reply:
Also this one is called package manager for 'YarnOrNpm', but others call it package installer in the var, I think package manager is more appropriate.